### PR TITLE
World mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/server/world/ServerWorld.mapping [hunk 1]
```diff
@@ -1,30 +1,61 @@
...
+		ARG 2 z
+		ARG 3 create
+	METHOD a addLightning (Lapj;)Z
 	METHOD a init (Lbcy;)V
 		ARG 1 levelInfo
+	METHOD a unloadBlockEntity (Lbow;)V
+	METHOD a unloadEntities (Ljava/util/Collection;)V
 	METHOD a save (Lzg;ZZ)V
 		ARG 1 progressListener
+	METHOD ag resetRain ()V
```
Comment (asiekierka) [inline mappings/net/minecraft/server/world/ServerWorld.mapping]: You sure?

File: mappings/net/minecraft/server/world/ServerWorld.mapping [hunk 2]
```diff
@@ -1,30 +1,61 @@
...
 	METHOD aj shouldSpawnAnimals ()Z
 	METHOD ak sendBlockActions ()V
+	METHOD b isInsideTick ()Z
 	METHOD b setChunkForced (IIZ)Z
 		ARG 1 x
 		ARG 2 z
-		ARG 3 force
+		ARG 3 forced
 	METHOD d getScoreboard ()Lqs;
+	METHOD e onSkippedToNextMorning ()V
```
Comment (asiekierka) [inline mappings/net/minecraft/server/world/ServerWorld.mapping]: Just onSkippedToMorning?

File: mappings/net/minecraft/world/chunk/Chunk.mapping [hunk 3]
```diff
@@ -1,4 +1,5 @@
 CLASS bsg net/minecraft/world/chunk/Chunk
+	METHOD a getHighestNonemptySection ()Lbst;
```
Comment (asiekierka) [inline mappings/net/minecraft/world/chunk/Chunk.mapping]: NonEmpty

File: mappings/net/minecraft/server/world/ServerWorld.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/server/world/ServerWorld.mapping]:
```java
    private void resetRain() {
        properties.setRainTime(0);
        properties.setRaining(false);
        properties.setThunderTime(0);
        properties.setThundering(false);
    }
```

Comment (asiekierka) [reply mappings/net/minecraft/server/world/ServerWorld.mapping]: resetWeather

File: mappings/net/minecraft/server/world/ServerWorld.mapping (hunk 2 above)
Comment (Runemoro) [reply mappings/net/minecraft/server/world/ServerWorld.mapping]: Done

File: mappings/net/minecraft/world/chunk/Chunk.mapping (hunk 3 above)
Comment (Runemoro) [reply mappings/net/minecraft/world/chunk/Chunk.mapping]: Done

File: mappings/net/minecraft/server/world/ServerWorld.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/server/world/ServerWorld.mapping]: Done

File: mappings/net/minecraft/client/MinecraftClient.mapping [hunk 4]
```diff
@@ -136,6 +136,7 @@ CLASS cqg net/minecraft/client/MinecraftClient
 	METHOD aB drawProfilerResults ()V
 	METHOD aC doAttack ()V
 	METHOD aD doItemUse ()V
+	METHOD aE handleEvents ()V
```
Comment (asiekierka) [inline mappings/net/minecraft/client/MinecraftClient.mapping]: What kind of events?

**Pull request merged**